examples/rap_blocksworld/inference.py

```diff
@@ -80,6 +80,7 @@ def rap_bw(base_model: LanguageModel,
 if __name__ == '__main__':
     import os
     import sys
+    sys.path.append('/data/haotian/RAP_tune/llm-reasoners/exllama')
```

Comment:
I believe this should be an argument? We don't want users to revise the file when running it

Reply:
Consider putting this in reasoners/llm/exllama_model.py ; Consider use relative path (the file path can be retrieved using __file__ and module path can be retrieved using reasoners.__file__)

Reply:
also, we may want to check whether setup.py can install exllama if user runs `pip install git+https://github.com/Ber666/llm-reasoners`. We may want to put the submodule inside reasoners folder.

Reply:
and please make sure everything is clear in the README if the user need to manually set up the module

Reply:
exllama has not create a setup.py so I think we cannot fix through pip install. 

> Consider putting this in reasoners/llm/exllama_model.py since we want to make it as convenient as possible for users; Consider using relative path (the file path can be retrieved using `__file__` and module path can be retrieved using `reasoners.__file__`)

I'll try to fix like this